steps/buildnumberadjust/1.0.0/step.yml

```diff
@@ -0,0 +1,45 @@
+title: Adjust BuildNumber
+summary: Adjust the $BITRISE_BUILD_NUMBER
+description: |-
+  In some cases you want to use the $BITRISE_BUILD_NUMBER for setting version info, but for some reason
+  you want to add, substract or just change the variable (for example when migrating from another build system
+  and you already have a certain build number to start with)
+website: https://github.com/stefandevo/bitrise-buildnumberadjust-step
+source_code_url: https://github.com/stefandevo/bitrise-buildnumberadjust-step
+support_url: https://github.com/stefandevo/bitrise-buildnumberadjust-step/issues
+published_at: 2017-04-30T19:59:35.758143112+02:00
+source:
+  git: https://github.com/stefandevo/bitrise-buildnumberadjust-step.git
+  commit: 4188b654c2ff941f2297cbecc9bf35cb148fd76b
+type_tags:
+- script
```

Comment:
@stefandevo I have an internal one, waiting for @godrei for the public one. Not sure if that was already published - we're in the middle of normalizing the tags, for better step discovery (main categories will be used on the UI too, soon ;))